tests/sql/environments/rac/debezium/perf/validator.py

```
@@ -0,0 +1,228 @@
+
+def main():
+    parser = argparse.ArgumentParser(description='Real-time OLR vs LogMiner validator')
+    parser.add_argument('--logminer', required=True, help='Path to logminer.jsonl')
+    parser.add_argument('--olr', required=True, help='Path to olr.jsonl')
+    parser.add_argument('--match-window', type=int, default=DEFAULT_MATCH_WINDOW,
+                        help=f'Seconds to wait for matching event (default: {DEFAULT_MATCH_WINDOW})')
+    parser.add_argument('--no-stop-on-fail', dest='stop_on_fail', action='store_false', default=True,
+                        help='Do not stop swingbench on mismatch')
+    args = parser.parse_args()
+
+    print(f'Validator starting', flush=True)
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)